packages/components/pages/f-contact-preferences/src/services/mapping/contactPreferences.mapper.js

```diff
@@ -0,0 +1,56 @@
+import {
+    NON_EDITABLE_PREFERENCE_KEYS,
+    NON_VISIBLE_PREFERENCE_KEYS
+} from '../../constants';
+
+export const mapToPreferencesViewModel = ({ Preferences, PreferenceVersionViewed }) => {
```

Comment:
Just a little unclear why the arguments are wrapped in curlies `{ Preferences, PreferenceVersionViewed }`

Reply:
Possibly JSDoc comments can help

Reply:
Object destructuring (i.e. a single object is passed and I extract & assign in a single line). I will add a comment 👍

Reply:
Thank you for explaining and adding the JSDocs. 
Would you take a suggestion that may make the intent a little clearer in my view. Feel free to object if you feel strongly about that.
 
```suggestion
export const mapToPreferencesViewModel = (preferencesResponseData) => {
    const { Preferences, PreferenceVersionViewed } = preferencesResponseData; 
```

Reply:
Just throwing in my 2-cents! Personally, I find the code is clearer with the inline destructure as it is currently (I do it in menu in quite a few places), I guess it's down to personal preference?

Reply:
Happy to approve if that is the consensus here. Thank you @DamianMullins